src/Shortcuts/Shortcut-item.scss

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+.shortcut-item {
+    margin-top: 10px;
+    .shortcut-span {
+        background-color: #cfd8dc;
```

Comment:
Use color defined in `styles.scss`

Reply:
Will go through design with Ketil before final PR